packages/client/modules/meeting/components/MeetingCheckInPrompt/NewCheckInQuestion.tsx

```diff
@@ -37,30 +37,6 @@ const CogIcon = styled('div')({
   width: 24
 })
 
-const QuestionBlock = styled('div')({
-  alignContent: 'center',
-  alignItems: 'center',
-  display: 'flex',
-  flexDirection: 'column',
-  fontSize: 24,
-  lineHeight: 1.25,
```

Comment:
Nit: I would bump this line-height (1.25) from 30px to 32px. 30px is a little too tight to my eyes. 36 is too much. Might at well use `px` units with `fontSize: 24`.